src/main/target/OMNIBUSF7/BetaflightF7.mk

```diff
@@ -0,0 +1 @@
+#BETAFLIGHTF7 Target file
```

Comment:
Inconsistent casing. Filename should be all caps to follow the usual BF target naming std.
```
$ make BETAFLIGHTF7
make: *** No rule to make target 'BETAFLIGHTF7'.  Stop.
$ make BetaflightF7
Building BetaflightF7
-----
```

Reply:
Indeed i dun goofed there. i did fix it but somehow wasn't picked up on git

Reply:
On Windows perhaps? Delete, commit, re-add with correct casing, commit. 
Or use a decent OS.

Reply:
That sounds probably - I've seen this kind of sh*ttyness with case sensitive names with RCS under Windows before.

Reply:
Such issues even occur on macOS sometimes. We had an entire iOS/Android project with similar issues because people got used to case-insensitive HFS :-(